tests/ganache/test_uniswap_v2_routing.py

```diff
@@ -899,7 +905,7 @@ def test_stateful_routing_two_legs(
 
     state.start_trades(datetime.datetime.utcnow(), trades)
     routing_model.execute_trades_internal(pair_universe, routing_state, trades, check_balances=True)
-    broadcast_and_resolve(web3, state, trades, stop_on_execution_failure=True)
+    execution_model.broadcast_and_resolve(state, trades, stop_on_execution_failure=True)
```

Comment:
broadcast_and_resolve should not have anything execution model specific.

No need to make this function part of execution model class.